mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/UidSearchAtoms.test

```diff
@@ -632,7 +632,8 @@ C: Rhubarb!
 S: \* 9 EXISTS
 S: \* 9 RECENT
 S: A72 OK (\[.+\] )?APPEND completed\.
-C: A73 APPEND testmailbox {11976+}
+
```

Comment:
I disagree with such a modification. It brakes downward test compatibility, and weakens tests.